styles/output/organic-chemistry-pdf.css

```diff
@@ -2039,13 +2047,13 @@ nav#toc > ol > li.os-toc-chapter > ol > li > a::after {
   line-height: 1.5rem;
 }
 
-.os-figure:not(.has-splash) > figure {
+.os-figure:not(.has-splash):not(.has-mechanism-figure) > figure {
```

Comment:
Are all styles rebuilt? I think this change should be visible in all Carnival books :thinking:

Reply:
@wilkus27 Thank you for noticing it! I forgot to regenerate all styles :disappointed: